tests/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
If we set the `convertDeprecationsToExceptions` to `false` in `phpunit.xml.dist` we might not need this file at all.